trace/opencensus/trace/labels_helper.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opencensus.trace.reporters import google_cloud_reporter
+
+import os
+
+# Environment variable set in App Engine when vm:true is set.
+_APPENGINE_FLEXIBLE_ENV_VM = 'GAE_APPENGINE_HOSTNAME'
+
+# Environment variable set in App Engine when env:flex is set.
+_APPENGINE_FLEXIBLE_ENV_FLEX = 'GAE_INSTANCE'
+
+_GAE_PROJECT_ENV = 'GCLOUD_PROJECT'
+_GAE_SERVICE_ENV = 'GAE_SERVICE'
+_GAE_VERSION_ENV = 'GAE_VERSION'
+
+
+class StackdriverLabels(object):  # pragma: NO COVER
+    """Stackdriver Trace common labels"""
+    AGENT = '/agent'
+    COMPONENT = '/component'
+    ERROR_MESSAGE = '/error/message'
+    ERROR_NAME = '/error/name'
+    HTTP_CLIENT_CITY = '/http/client_city'
+    HTTP_CLIENT_COUNTRY = '/http/client_country'
+    HTTP_CLIENT_PROTOCOL = '/http/client_protocol'
+    HTTP_CLIENT_REGION = '/http/client_region'
+    HTTP_HOST = '/http/host'
+    HTTP_METHOD = '/http/method'
+    HTTP_REDIRECTED_URL = '/http/redirected_url'
+    HTTP_REQUEST_SIZE = '/http/request/size'
+    HTTP_RESPONSE_SIZE = '/http/response/size'
+    HTTP_STATUS_CODE = '/http/status_code'
+    HTTP_URL = '/http/url'
+    HTTP_USER_AGENT = '/http/user_agent'
+    PID = '/pid'
+    STACKTRACE = '/stacktrace'
+    TID = '/tid'
+
+
+class GAELabels(object):  # pragma: NO COVER
+    """GAE common labels"""
+    GAE_VERSION = 'g.co/gae/app/version'
```

Comment:
Perhaps this could be a dictionary?

```
# Mapping from environment variable to resource label
GAE_FLEX_LABELS = {
  'GAE_VERSION': 'g.co/gae/app/version',
  ...
}

for env_var, label_name in GAE_FLEX_LABELS.items():
  value = os.environ.get(env_var)
  if value:
    self.tracer.add_label_to_spans(label_name)
```